swarm/version/version.go

```diff
@@ -59,6 +64,7 @@ func VersionWithCommit(gitCommit string) string {
 	vsn := Version
 	if len(gitCommit) >= 8 {
 		vsn += "-" + gitCommit[:8]
+		GitCommit = gitCommit[:8]
```

Comment:
This is redundant. We can just use this function, it also gives the current version, next to the commit, which is not bad.